cache/compressed_secondary_cache.cc

```diff
@@ -56,24 +58,26 @@ std::unique_ptr<SecondaryCacheResultHandle> CompressedSecondaryCache::Lookup(
   Status s;
 
   if (cache_options_.compression_type == kNoCompression) {
-    s = create_cb(ptr->get(), cache_->GetCharge(lru_handle), &value, &charge);
+    s = create_cb(ptr->get(), handle_value_charge, &value, &charge);
   } else {
     UncompressionContext uncompression_context(cache_options_.compression_type);
     UncompressionInfo uncompression_info(uncompression_context,
                                          UncompressionDict::GetEmptyDict(),
                                          cache_options_.compression_type);
 
+    RecordTick(stats, COMP_SEC_CACHE_BYTES_COMPRESSED, handle_value_charge);
```

Comment:
Why record stats here and not in Insert? A lot of blocks may get evicted without any lookup.

Reply:
> Why record stats here and not in Insert? A lot of blocks may get evicted without any lookup.

1. Lookup carries the parameter statistics, but Insert doesn't.
2. It may also benefit when Insert once and Lookup multiple times in future.